stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,929 @@
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br /><br />
+`;
+
+tablet.storyName = 'Tablet mode';
+tablet.parameters = {
+    docs: {
+        iframeHeight: 150,
+        storyDescription: 'The dynamic page with Tablet specification(L). Use the `--m` modifier on `title-area`, `breadcrumb`, ' +
+        '`header`, and `content` elements to get the Tablet spacings.' +
+        ' The spec recommends navigation actions in a separate row above the global actions. The navigation actions are placed at the same level as the breadcrumb.'
+    }
+};
+
+export const mobile = () => `
+<div class="mobile-container">
+    <div class="fd-dynamic-page" role="region" aria-label="Dynamic Page Mobile Example">
+        <div class="fd-dynamic-page__title-area fd-dynamic-page__title-area--s" tabindex="0">
+            <div class="fd-dynamic-page__breadcrumb-container">
+                <ul class="fd-dynamic-page__breadcrumb--s fd-breadcrumb">
+                    <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Men</a></li>
+                    <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Shoes</a></li>
+                    <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Running Shoes</a></li>
+                </ul>
+                <div class="fd-dynamic-page__toolbar fd-dynamic-page__toolbar--actions fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+                        <button class="fd-button fd-button--transparent" aria-label="Close">
+                            <i class="sap-icon--decline"></i>
+                        </button>
+                </div>
+            </div>
+            <div class="fd-dynamic-page__title-container">
+                <div class="fd-dynamic-page__title"> Balenciaga Tripple S Trainers </div>
+                    
+                    <div class="fd-dynamic-page__toolbar fd-dynamic-page__toolbar--actions fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+                        <button class="fd-button fd-button--transparent" aria-label="More Header Content">
+                            <i class="sap-icon--overflow"></i>
+                        </button>
+                        <button class="fd-button fd-button--transparent" aria-label="Header actions">
+                            <i class="sap-icon--overflow"></i>
+                        </button>
+                    </div>
+            </div>
+            <div class="fd-dynamic-page__subtitle"> Expanded header in phone </div>
```

Comment:
stick to the term mobile, instead of phone

Reply:
fixed